libraries/botbuilder-ai/botbuilder/ai/luis/luis_recognizer_options_v3.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+
+from botbuilder.core import BotTelemetryClient, NullTelemetryClient
+from .luis_recognizer_options import LuisRecognizerOptions
+
+
+class LuisRecognizerOptionsV3(LuisRecognizerOptions):
+    def __init__(
+        self,
+
+        include_all_intents: bool = False,
+        include_instance_data: bool = True,
+        log: bool = True,
+        prefer_external_entities: bool = False,
```

Comment:
I note that JS defaults 'prefer_external_entities' to True.

Reply:
Fixed